resources/lang/hi/ranking.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+return [
+    'title' => 'श्रेणी',
+    'table' => [
+        'rank' => 'पद',
+        'charname' => 'व्यक्ति का नाम',
+        'job-name' => 'काम',
+        'guild' => 'समाज',
+        'level' => 'स्तर',
+        'job-type' => 'नौकरी के प्रकार',
+        'trader' => 'व्यापारी',
+        'hunter' => 'शिकारी',
+        'thief' => 'चोर',
+        'itempoints' => 'आइटम के लिए अंक',
+        'exp' => 'अनुभव',
+        'job-level' => ':exp अनुभव स्तर :level के साथ',
+    ],
+    'no-player' => 'उस सर्वर पर कोई व्यक्ति नहीं मिला',
+    'no-guild' => 'उस सर्वर पर कोई समाज नहीं मिला',
+    'no-job' => 'उस सर्वर पर कोई जॉब कैरेक्टर नहीं मिला',
+
+    'search' => [
+        'placeholder' => 'व्यक्ति का नाम, समाज, काम',
+        'filter-by' => 'के साथ छान लें',
+        'charname' => 'व्यक्ति',
+        'guild' => 'समाज',
+        'jobname' => 'काम',
+        'trader' => 'व्यापारी',
+        'hunter' => 'शिकारी',
+        'thief' => 'चोर',
+        'submit' => 'खोज'
```

Comment:
We are missing the search-parameters.

```
// do not change this
        'search-charname' => 'charname',
        'search-guild' => 'guild',
        'search-job' => 'job',
        'search-trader' => 'trader',
        'search-hunter' => 'hunter',
        'search-thief' => 'thief',
```

they are needed for the search in the Controller.

Reply:
I'm aware of this and [mentioned it](https://github.com/Devsome/silkroad-laravel/pull/80#issue-502243135). I removed them because [you have specified a fallback locale in the application](https://github.com/Devsome/silkroad-laravel/blob/master/config/app.php#L93-L104) or maybe you just left it as default when you installed laravel the first time. And Laravel internally falls back to [this default language](https://github.com/illuminate/translation/blob/master/Translator.php#L122) when the key is not present. So it would be redundant to add.

However, if you feel you still need this, I can make a commit to this PR adding them back.

Reply:
Yeah my fault, I'm going to use this in a config. Thanks for making this clear

Reply:
No problem. It's nobody's fault--just that Laravel is too awesome 👍 

Kindly add the `hacktoberfest-accepted` before accepting this PR as I would like it to count towards Hacktoberfest. Thank you and have a nice day.